airbyte-cdk/python/unit_tests/sources/file_based/test_scenarios.py

```diff
@@ -87,16 +95,20 @@
     csv_custom_format_scenario,
     multi_stream_custom_format,
     empty_schema_inference_scenario,
+    single_parquet_scenario,
+    multi_parquet_scenario,
+    parquet_various_types_scenario,
 ]
 
 
 @pytest.mark.parametrize("scenario", discover_scenarios, ids=[s.name for s in discover_scenarios])
-def test_discover(capsys, tmp_path, json_spec, scenario):
+def test_discover(capsys: CaptureFixture[str], tmp_path: PosixPath, json_spec: Mapping[str, Any], scenario: TestScenario) -> None:
```

Comment:
added these type annotations as part of https://github.com/airbytehq/airbyte/pull/27790